civiform/civiform
# Back button in Edit TI Group Members View

### Description

Adding a 'Back' button to the edit TI group members flow, allowing users to easily go back to the previous page.

### Checklist

#### General


#### User visible changes


### Instructions for manual testing

DevTools -> Program and CiviForm Admin -> Intermediaries
- Create a TI group, then, click on 'Edit members'
- See if 'Back' button is available and check if it returns you to the previous page.

### Issue(s) this completes

Fixes #1426
